department-of-veterans-affairs/va.gov-cms
# VACMS-15085: Eevents can break content build

## Description
Adds a new constraint to the event date field which prevents saving an event if it

Relates to #15085

## Testing done
Manual testing

## Screenshots
<img width="1106" alt="Screenshot 2023-09-06 at 1 57 23 PM" src="https://github.com/department-of-veterans-affairs/va.gov-cms/assets/221539/e8bf6bf7-4f54-4a17-94fe-21b854696930">

## QA steps
Set recurring: recur every day for 7 days, until 10/7/2023
Save draft
1. Create an event: 10/1/2023, 8-9a
2. Set recurring: recur every day for 7 days, until 10/7/2023
3. Save draft
4. Edit draft
5. Open Edit recurrences window
6. Dropdown arrow on each recurrence -> Remove recurrence
7. Save draft
   - [x] Validate an error message appears that read: 'Events cannot be created without a start and end date.'

### Definition of Done

### Select Team for PR review
